android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -823,27 +823,27 @@ private Integer getPidForProcessName(String processName) {
     	for (int x = 0; x < headers.length; x++) {
     		if(headers[x].equals("PID")) {
     			PidIndex = x;
-    			continue;
+    			break;
     		}
     	}
 		if(Logging.DEBUG) Log.d(Logging.TAG,"getPidForProcessName(): PID at index: " + PidIndex + " for output: " + processLinesAr[0]);
     	
 		Integer pid = null;
     	for(int y = 1; y < processLinesAr.length; y++) {
-    		Boolean found = false;
+    	    Boolean found = false;
     	    String [] comps = processLinesAr[y].split("[\\s]+");
     	    for(String arg: comps) {
     	    	if(arg.equals(processName)) {
     	    		if(Logging.DEBUG) Log.d(Logging.TAG,"getPidForProcessName(): " + processName + " found in line: " + y);
     	    		found = true;
+                    break;
```

Comment:
Please use the same indentation level or adjust the previous two lines as well. Otherwise ok.